test/remote/gateways/remote_orbital_test.rb

```diff
@@ -1301,6 +1321,13 @@ def test_successful_purchase_with_level_2_data
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_with_level_2_data_canadian_currency
+    response = @tandem_gateway.purchase(@amount, @credit_card, @options.merge(currency: 'CAD', merchant_vat_reg_number: '987654321', national_tax: '625', level_2_data: @level_2_options))
```

Comment:
Small q: should the `merchant_vat_reg_number` and `national_tax` be nested in the `level_2_data`? It doesn't look like we access them from top level `options`

It looks like that's what you did up in lines 54-62; although I'm not quite clear on the difference between that version of `level_2_options` and the one used in this section 🤔

Reply:
These fields should only be used by Canadian merchants so they are not included in the main options hash.